pkg/server/worker.go

```diff
@@ -12,15 +12,27 @@ import (
 func (s *Server) worker(name string, h *host.Host) {
 	defer s.wg.Done()
 
+	// Add a random delay of 1-59 seconds before starting to reduce initial filesystem activity
```

Comment:
once the host count got high enough on a low end cloud instance, initial rrd initialization in parallel cause the system to have a stroke